certbot/ocsp.py

```diff
@@ -77,22 +117,38 @@ def determine_ocsp_server(self, cert_path):
         :returns: (OCSP server URL or None, OCSP server host or None)
 
         """
-        try:
-            url, _err = util.run_script(
-                ["openssl", "x509", "-in", cert_path, "-noout", "-ocsp_uri"],
-                log=logger.debug)
-        except errors.SubprocessError:
-            logger.info("Cannot extract OCSP URI from %s", cert_path)
-            return None, None
+        if self.use_openssl_binary:
+            try:
+                url, _err = util.run_script(
+                    ["openssl", "x509", "-in", cert_path, "-noout", "-ocsp_uri"],
```

Comment:
You should be able to able to extract this data from the cert with cryptography, even if it's an older release.

Reply:
Minimal cryptography version to get the OCSP OIDs is 0.8, and Certbot requires at least 1.2. So we are covered: code is simplified to use always cryptography here.